proto/sentry_protos/snuba/v1/endpoint_aggregate_bucket.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
```

Comment:
Due to the TODO, we will have breaking api changes to this rpc to support multiple y axis and group bys. Not sure what else yet but just wanted to call this out.

Reply:
Added